worker/detectors/feature/dpkg/dpkg.go

```diff
@@ -74,28 +78,34 @@ func (detector *DpkgFeaturesDetector) Detect(data map[string][]byte) ([]database
 
 			pkg.Feature.Name = md["name"]
 			if md["version"] != "" {
-				pkg.Version, err = types.NewVersion(md["version"])
+				version := md["version"]
+				err = versionfmt.Valid("dpkg", version)
```

Comment:
Seems we hardcore "dpkg" and "rpm" everyone. Should it be a constant in the parser?